src/chip.tsx

```diff
@@ -27,7 +28,8 @@ const useStyles = createUseStyles(({colors, mq}) => ({
         '& > span': {
             color: colors.textPrimary, // Giving color to text
         },
-
+    },
+    interactive: {
```

Comment:
can we remove hover state only when a chip is selected?

Reply:
I don't understand this

Reply:
it is quite weird interaction removing border in hover state when a chip is selected. Keep the hover behavior but not the style that remove the border

https://user-images.githubusercontent.com/6722153/158561686-2608a846-63e3-4afe-b0d6-f15f7b8e9a64.mov

Reply:
ok, try now

Reply:
perfect ✨